src/Feature/Catalog/code/Wooli.Feature.Catalog.csproj

```diff
@@ -62,6 +62,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="AutoMapper, Version=6.2.2.0, Culture=neutral, PublicKeyToken=be96cd2c38ef1005, processorArchitecture=MSIL">

```

Comment:
Please make sure this version is used in all other projects.

Reply:
Yes, we use 6.2.2 version in our other projects